src/config/dynamic-theme-fixes.config

```diff
@@ -3516,6 +3516,16 @@ CSS
 
 ================================
 
+kinoart.ru
+
+INVERT
```

Comment:
All are generated and likely to change in the future on a predictable basis.
The best is to use DOM based selector e.g. for `._2nHMA`
```CSS
main > div > svg`
```

Reply:
As I understand it, there shouldn't be a "\`" at the end. But even without it, this rule only works for the logo next to the header. In this case, the logo next to the footer is not fixed, and on the second page the logos next to the header are not fixed either. I tried `svg`, but then other elements change too, which shouldn't change.

Reply:
It was rather a example to use the dom based selectors. As like I said using the selector can break in a near future.

Reply:
I tried to pick a DOM based selector myself that would only invert the logos, but I couldn't. If you can find one, it will be great.

Reply:
```CSS

kinoart.ru

INVERT
main > div > svg
footer > div > svg
header > div > a > svg
```

?

Reply:
Thanks. And what is the `main` tag, because it is not in the page code? What is it used for?

Reply:
![](https://i.imgur.com/B2qSpup.png)

Reply:
Didn't notice. Thanks again!